package.json

```diff
@@ -1,55 +1,20 @@
+    "test": "buidler test"
+  },
   "dependencies": {
     "@aragon/os": "^4.3.0"
   },
   "devDependencies": {
-    "@aragon/apps-shared-minime": "^1.0.2",
-    "@aragon/apps-token-manager": "2.1.0",
-    "@aragon/apps-voting": "2.1.0",
-    "@aragon/cli": "^7.0.1",
-    "@aragon/templates-shared": "^1.0.1",
-    "@aragon/test-helpers": "^2.0.0",
-    "@aragon/truffle-config-v5": "^1.0.0",
-    "babel-eslint": "^10.0.2",
-    "cross-env": "^5.2.0",
-    "eslint": "^6.0.1",
-    "eslint-config-prettier": "^6.0.0",
-    "eslint-config-standard": "^13.0.1",
-    "eslint-config-standard-react": "^8.0.0",
-    "eslint-plugin-import": "^2.18.0",
-    "eslint-plugin-node": "^9.1.0",
-    "eslint-plugin-prettier": "^3.1.0",
-    "eslint-plugin-promise": "^4.2.1",
-    "eslint-plugin-react": "^7.14.2",
-    "eslint-plugin-standard": "^4.0.0",
-    "ethlint": "^1.2.4",
-    "fs-extra": "^8.1.0",
-    "ganache-cli": "^6.0.0",
-    "prettier": "^1.18.2",
-    "solidity-coverage": "^0.6.2",
-    "truffle": "^5.1.4"
-  },
-  "scripts": {
-    "prepare": "cd app && npm install && cd ..",
-    "start": "npm run start:ipfs",
-    "start:ipfs": "aragon run --files dist",
-    "start:http": "aragon run --http localhost:8001 --http-served-from dist",
-    "start:ipfs:template": "aragon run --files dist --template template --template-init 0x5d94e3e7aec542ab0f9129b9a7badeb5b3ca0f77 @ARAGON_ENS 0xd526b7aba39cccf76422835e7fd5327b98ad73c9 0xf1f8aac64036cdd399886b1c157b7e3b361093f3 --template-new-instance newTokenAndInstance --template-args MyToken TKN ['\"0xb4124cEB3451635DAcedd11767f004d8a28c6eE7\"'] ['\"1000000000000000000\"'] ['\"500000000000000000\",\"150000000000000000\",\"86400\"']",
-    "start:http:template": "aragon run --http localhost:8001 --http-served-from dist --template template --template-init 0x5d94e3e7aec542ab0f9129b9a7badeb5b3ca0f77 @ARAGON_ENS 0xd526b7aba39cccf76422835e7fd5327b98ad73c9 0xf1f8aac64036cdd399886b1c157b7e3b361093f3 --template-new-instance newTokenAndInstance --template-args MyToken TKN ['\"0xb4124cEB3451635DAcedd11767f004d8a28c6eE7\"'] ['\"1000000000000000000\"'] ['\"500000000000000000\",\"150000000000000000\",\"86400\"']",
-    "start:app": "cd app && npm start && cd ..",
-    "build": "cd app && npm run build && cd ..",
-    "lint": "eslint . & solium --dir ./contracts",
-    "lint:fix": "eslint . --fix & solium --dir ./contracts --fix",
-    "compile": "aragon contracts compile",
-    "test": "cross-env TRUFFLE_TEST=true npm run ganache-cli:test",
-    "coverage": "cross-env SOLIDITY_COVERAGE=true npm run ganache-cli:test",
-    "ganache-cli:test": "sh ./node_modules/@aragon/test-helpers/ganache-cli.sh",
-    "publish:patch": "aragon apm publish patch",
-    "publish:minor": "aragon apm publish minor",
-    "publish:major": "aragon apm publish major",
-    "versions": "aragon apm versions"
+    "@aragon/buidler-aragon": "^0.0.1-beta.10",
+    "@aragon/contract-test-helpers": "0.0.1",
+    "@nomiclabs/buidler": "^1.0.2"
```

Comment:
Do we not need to specify a truffle config with buidler? Does our own plugin handle this? How do I eject?

Reply:
The equivalent of the truffle config file in Buidler is the Buidler config file. It allows you to define networks, solc params, and plugin specific configuration variables.

Reply:
What do you mean by eject? Like in create-react-app eject? What do you imagine would happen here if that command existed?

Reply:
> The equivalent of the truffle config file in Buidler is the Buidler config file

Is this something we control in our plugin or is this a buidler-level thing?

> What do you mean by eject? Like in create-react-app eject? What do you imagine would happen here if that command existed?

Really just related to the previous question; what if I don't like the settings and want to change them? I imagine if the config we're using is at the buidler-level, then it's fairly transparent but wanted to make sure in case it was our own plugin's config.

Reply:
Oh, the plugin doesn't have a buidler.config.js. Projects that use the plugin do. Like in this case, the aragon-react-boilerplate has one.